src/abstractdataframe/show.jl

```diff
@@ -602,7 +602,12 @@ function _show(io::IO,
         crop = :both
     end
 
-    compact_printing::Bool = get(io, :compact, true)
+    # For consistency, if `kwargs` has `compact_printng`, we must use it.
+    if haskey(kwargs, :compact_printing)
+        compact_printing = kwargs[:compact_printing]
+    else
+        compact_printing::Bool = get(io, :compact, true)
+    end
```

Comment:
Or maybe even this? :-)
```suggestion
        compact_printing::Bool = get(kwargs, :compact_printing, get(io, :compact, true))
```

Reply:
This is awesome!

Reply:
I hope it works, I haven't checked. :-p

Reply:
seems OK